internal/condition/eval/eval.go

```diff
@@ -2,13 +2,29 @@ package eval
 
 import (
 	"fmt"
+	"time"
 
 	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"github.com/openfga/openfga/internal/build"
 	"github.com/openfga/openfga/internal/condition"
+	"github.com/openfga/openfga/internal/server/config"
+	"github.com/openfga/openfga/internal/utils"
 	"github.com/openfga/openfga/pkg/typesystem"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	"google.golang.org/protobuf/types/known/structpb"
 )
 
+var conditionEvaluationCostHistogram = promauto.NewHistogram(prometheus.HistogramOpts{
+	Namespace:                       build.ProjectName,
+	Name:                            "condition_evaluation_cost",
+	Help:                            "A histogram of the CEL evaluation cost of a Condition in a Relationship Tuple",
+	Buckets:                         utils.LinearBuckets(1, config.DefaultMaxConditionEvaluationCost, 10),
```

Comment:
```suggestion
	Buckets:                         utils.LinearBuckets(0, config.DefaultMaxConditionEvaluationCost, 10),
```

since I thought some expressions could have cost 0?

Reply:
https://github.com/openfga/openfga/pull/1235#discussion_r1428250705